docs/configuring.md

```diff
@@ -0,0 +1,180 @@
+
+Limits are controlled by the **ulimit** command and are referred to as a *hard* limit when set by the **ROOT** user, and a *soft* limit when set by other users. To see the *soft* limits set for a system, run `ulimit -a`. The output is similar to the following example:
+
+```
+core file size          (blocks, -c) 0
+data seg size           (kbytes, -d) unlimited
+file size               (blocks, -f) unlimited
+max locked memory       (kbytes, -l) unlimited
+max memory size         (kbytes, -m) unlimited
+open files                      (-n) 256
+pipe size            (512 bytes, -p) 1
+stack size              (kbytes, -s) 8192
+cpu time               (seconds, -t) unlimited
+max user processes              (-u) 2784
+virtual memory          (kbytes, -v) unlimited
+```
+
+To show *hard* limits, use `ulimit -Ha`.
+
+You can change limits for specific resources on a temporary basis by running the **ulimit** command. Alternatively, you can store limit settings in a configuration file, which is `/etc/security/limits` for AIX or `etc/security/limits.conf` for Linux. For more information about configuring resource limits, refer to the documentation for your operating system.   
+
+The main use case for changing resource limits is when enabling a system dump to ensure that all the required data can be collected for analysis. For more information, see [Enabling a full system dump](dump_systemdump.md#enabling-a-full-system-dump-aix-and-linux).
+
+## Setting resource limits (z/OS)
+
+Resource limits imposed by z/OS might affect Java operations. To learn how these resource limits are set, see [Customizing the BPXPRMxx member of SYS1.PARMLIB](https://www.ibm.com/support/knowledgecenter/SSLTBW_2.4.0/com.ibm.zos.v2r4.bpxb200/cusis.htm).
+
+To see the current settings, enter the following z/OS operator command:
+
+```
+D OMVS,O
+```
+The suggested minimum values for Java applications are shown in the following table:
+
+| Parameter         | Value      |
+|-------------------|------------|
+|**MAXPROCSYS**     | 900        |
+|**MAXPROCUSER**    |	512        |
+|**MAXUIDS**	      | 500        |
+|**MAXTHREADS**     |	10000      |
+|**MAXTHREADTASKS**	| 5000       |
+|**MAXASSIZE**	    | 2147483647 |
+|**MAXCPUTIME**	    | 2147483647 |
+|**MAXMMAPAREA** 	  | 40960      |
+|**IPCSEMNIDS**	    | 500        |
+|**IPCSEMNSEMS**	  | 1000       |
+
+
+<i class="fa fa-pencil-square-o" aria-hidden="true"></i> **Notes:**
+
+1. The number of threads that can be created by a Java process is limited by the lower of the two values for **MAXTHREADS** and **MAXTHREADSTASKS**.
+2. The number of 4K pages that are available for memory-mapped `*.jar` files is limited by **MAXMAPAREA** through the **JAVA_MMAP_MAXSIZE** environment variable.
```

Comment:
I don't find any reference to JAVA_MMAP_MAXSIZE in OpenJ9/OMR or OpenJDK code. I expect this is an IBM Java 8 thing.